test/views/resident/properties/gain/WorthWhenGaveAwayViewSpec.scala

```diff
@@ -117,6 +114,21 @@ class WorthWhenGaveAwayViewSpec extends UnitSpec with WithFakeApplication with F
         "has the field name as 'amount' to bind correctly to the form" in {
 
         }
+
+        "have two paragraphs" which {
+          s"have a p tag with the text ${messages.paragraphText}" in {
+            form.select("p#guideText").text shouldBe messages.paragraphText
+          }
+
+          "have a p tag" which {
+            s"with the extra text ${messages.extraText}" in {
+              form.select("p.block-panel-border").text shouldBe messages.extraText
+            }
+            s"with the classes ' and the classes 'panel panel-border-wide'" in {
```

Comment:
Can you use these classes please : panel-indent panel-indent--wide panel-indent--gutter